Library/Homebrew/tap.rb

```diff
@@ -27,10 +27,10 @@ def self.fetch(*args)
     end
 
     # we special case homebrew so users don't have to shift in a terminal
-    user = "Homebrew" if user == "homebrew"
+    user = user.capitalize if ["homebrew", "linuxbrew"].include? user
     repo = repo.strip_prefix "homebrew-"
 
-    if user == "Homebrew" && (repo == "homebrew" || repo == "core")
```

Comment:
Does the `repo == "homebrew"` break anything here? If it's not required for the fix can you put it back? Thanks!

Reply:
It doesn't break anything. Just seemed to be an anachronism. I can put it back.